src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -101,6 +105,10 @@ public void save(AddressBook addressBook) throws StorageOperationException {
         } catch (JAXBException jaxbe) {
             throw new StorageOperationException("Error converting address book into storage format");
         }
+        
+        if(!fileExist){
+        	throw new StorageOperationException("Warning, your storage file may have been deleted, the current storage file has recovered what we could");
+        }
```

Comment:
- Good job in throwing the exception. But you have not handled it. You can not just throw exceptions and let someone else deal with it.

- Implement how to catch and handle this exception. Hint: Main.java executeCommand